scripts/extract_translation_templates.py

```diff
@@ -1,117 +1,134 @@
 #!/usr/bin/env python3
-
 import os
 import sys
-import subprocess
 import tempfile
 import logging
 import argparse
 import difflib
-
 import coloredlogs
-
+import requests
+import re
 from eq_translations.entrypoints import handle_extract_template
 
-logger = logging.getLogger(__name__)
+page = requests.get("https://github.com/ONSdigital/eq-translations/commit/master/")
 
-coloredlogs.install(level="DEBUG", logger=logger, fmt="%(message)s")
 
-SCHEMAS_TO_EXTRACT = [
-    "ccs_household_gb_eng",
-    "census_individual_gb_wls",
-    "census_individual_gb_nir",
-    "census_household_gb_wls",
-    "census_household_gb_nir",
-]
+def get_ref():
+    f = open("Pipfile.lock", "r")
+    file = f.readlines()
+    for i, line in enumerate(file):
+        if '"eq-translations"' in line:
+            ref = re.compile(r"\w{40}")
+            mo = ref.search(file[i + 3])
+            return mo.group()
+    print("eq-translations not in Pipfile.lock")
 
 
-def get_template_content(filename, ignore_context=False):
-    line_beginnings_to_ignore = ['"POT-Creation-Date']
+if get_ref() in page.text:
+    print("eq-translations up to date")
 
```

Comment:
We shouldn't indent it like this. This conditional should be done inside:
```
if __name__ == "__main__":
	...
```

Reply:
I moved it as requested, thanks for review!